src/internal/obj/obj_intgtest/tests.go

```diff
@@ -0,0 +1,83 @@
+package obj_intgtest
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/obj"
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	tu "github.com/pachyderm/pachyderm/v2/src/internal/testutil"
+)
+
+// BackendType is used to tell the tests which backend is being tested so some
+// testing can be skipped for backends that do not support certain behavior.
+type BackendType string
+
+const (
+	AmazonBackend    BackendType = "Amazon"
+	ECSBackend       BackendType = "ECS"
+	GoogleBackend    BackendType = "Google"
+	MicrosoftBackend BackendType = "Microsoft"
+	LocalBackend     BackendType = "Local"
+)
+
+// ClientType is used to tell the tests which client is being tested so some testing
+// can be skipped for clients that do not support certain behavior.
+type ClientType string
+
+const (
+	AmazonClient    ClientType = "Amazon"
+	MinioClient     ClientType = "ECS"
+	GoogleClient    ClientType = "Google"
+	MicrosoftClient ClientType = "Microsoft"
+	LocalClient     ClientType = "Local"
+)
+
+// testInterruption
+// Interruption is currently not implemented on the Amazon, Microsoft, and Minio clients
+//  Amazon client - use *WithContext methods
+//  Microsoft client - move to github.com/Azure/azure-storage-blob-go which supports contexts
+//  Minio client - upgrade to v7 which supports contexts in all APIs
+//  Local client - interruptible file operations are not a thing in the stdlib
+func testInterruption(t *testing.T, client obj.Client) {
```

Comment:
Not that it works in the first place, but moving this here - this test will never run for local clients which seems incorrect to me - this behavior _should_ be supported by all clients in the long run.

Reply:
Also, this appears to have removed the `t.Skip()` logic for all the broken clients - are these tests just failing?  This should probably be run in a sub-test regardless, rather than calling this directly from the client test suite.

Reply:
I didn't call it for the broken clients. That is why it's not in the test suite.

Reply:
Hmmm - this is called by the microsoft client which doesn't support contexts - I wonder if the test is broken.

Reply:
Also, the old behavior was to have a `t.Skip()` which would provide some visibility into what tests were missing or broken and now the only indication is the comment above.  I'm not sure it's worth changing back at this point, but I think there was some value in it.